qa/tasks/cephfs/test_volumes.py

```diff
@@ -2753,7 +2753,7 @@ def test_subvolume_create_without_case_sensitivity(self):
     def test_subvolume_create_with_case_insensitive(self):
         # create subvolume
         subvolume = self._gen_subvol_name()
-        self._fs_cmd("subvolume", "create", self.volname, subvolume, "--case-insensitive")
+        self._fs_cmd("subvolume", "create", self.volname, subvolume, "--casesensitive=0")
```

Comment:
nit: should the test method name end with `case_sensitive` instead ?

Reply:
No, only '-' and '_' are interchangeable. '' is not one of those. :)

Reply:
by test method name I meant `def test_subvolume_create_with_case_insensitive(self):`

Reply:
oh I see.  Yes I guess but I don't think it's worth rolling the jenkins test again for that, no?

Reply:
oh, but anyway, for this specific test `--casesensitive=0` **is** case insensitive so I don't think the wording needs changed.